### PR TITLE
Add virtual network peering api

### PR DESCRIPTION
**Pull request by sharsonia**

### CODE REVIEW

File: lisa/sut_orchestrator/azure/common.py [hunk 1]
```diff
@@ -719,6 +720,56 @@ def delete_private_dns_zone_groups(
         log.debug(f"not find private dns zone group: {private_dns_zone_group_name}")
 
 
+def create_update_virtual_network_peerings(
```
Comment (squirrelsc) [inline lisa/sut_orchestrator/azure/common.py:723]: Please make it e2e work, and we can discuss and then enable it from a PR.

**Comment by LiliDeng** [comment]
@sharsonia if no plan to update it recently, please close this one, raise new one once it is ready for review, thanks!

**Comment by sharsonia** [comment]
> @sharsonia if no plan to update it recently, please close this one, raise new one once it is ready for review, thanks!

I am gonna have series of commits when this works e2e. Will soon add other commits too

**Comment by LiliDeng** [comment]
@sharsonia feel free to create new one or reopen it when it is ready, thanks

**Pull request closed without merging**